backend/core/test/authz/authz.e2e-spec.ts

```diff
@@ -31,7 +31,7 @@ describe("Authz", () => {
   let userAccessToken: string
   let adminAccessToken: string
   let userProfile: UserDto
-  const adminOnlyEndpoints = ["/preferences", "/units", "/translations"]
+  const adminOnlyEndpoints = ["/units", "/translations"]
```

Comment:
We had preferences here, but not programs - thoughts on what the permissions should be for all multiselect questions? @seanmalbert

Reply:
Read should be fully open and the rest of the perms should be locked to admin and jurisdictional admins (I think we want JAs to be able to create, edit and delete questions).